pkg/bpmn_engine/engine.go

```diff
@@ -184,6 +193,18 @@ func (state *BpmnEngineState) run(instance *ProcessInstanceInfo) (err error) {
 	return err
 }
 
+func (state *BpmnEngineState) findUnCompletedUserTasksForContinuation(process *ProcessInfo, instance *ProcessInstanceInfo) (ret []*BPMN20.TUserTask) {
+	for _, event := range process.definitions.Process.UserTasks {
+		for _, job := range state.jobs {
+			if (job.State == activity.Active || job.State == activity.Ready) && job.ProcessInstanceKey == instance.instanceKey && job.ElementId == event.GetId() {
```

Comment:
Assuming, we allow application developers to exit a handler without calling job.Complete() or job.Fail() ... we only have active jobs to continue.
I do propose:
1. rename the function -> findActiveUserTasksForContinuation
2. just search for job.State == activity.Active

Reply:
My earliest version just like this.